README.md

```diff
@@ -25,6 +25,8 @@
 ## About
 This Karma plugin helps testing your UI5 projects.
 
+Please refer to the [UI5 Testing Developer Guide](https://openui5.hana.ondemand.com/#/topic/7cdee404cac441888539ed7bfe076e57) for information on writing tests for your project.
```

Comment:
Please refer to the [Testing](link) section in the UI5 Developer Guide for information about writing tests for your project.

Reply:
Thanks!